packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/steps/Origine.tsx

```diff
@@ -0,0 +1,53 @@
+import { InputRadio } from "@socialgouv/cdtn-ui";
+import React from "react";
+import { Field } from "react-final-form";
+
+import { ErrorField } from "../../common/ErrorField";
+import { Question } from "../../common/Question";
+import { RadioContainer } from "../../common/stepStyles";
+import { required } from "../../common/validators";
+
+function OrigineStep() {
+  return (
+    <>
+      <Question as="p" required>
+        Qui est à l’origine du départ en retraite&nbsp;?
+      </Question>
+      <RadioContainer>
+        <Field
+          type="radio"
+          name="contrat salarié - mise à la retraite"
+          value="oui"
+          id="mise-retraite"
+          validate={required}
+        >
+          {(props) => (
+            <InputRadio
+              id="mise-retraite"
+              label="L’employeur décide de lui-même de mettre à la retraite le salarié par une décision adressée à celui-ci."
+              {...props.input}
+            />
+          )}
+        </Field>
+        <Field
+          type="radio"
+          name="contrat salarié - mise à la retraite"
+          value="non"
+          id="depart-retraite"
```

Comment:
not sure id is a prop of `<Field>` component

Reply:
I deleted them. I don't know why I put an `id` here 🤷